client/src/pages/Modal.scss

```diff
@@ -0,0 +1,62 @@
+.modal {
```

Comment:
could we move `Modal.scss` to `components/organisms/style`? since `Modal.tsx` is in `components/organisms`